brands/amenity/restaurant.json

```
@@ -2706,6 +2706,18 @@
       "name": "Zizzi"
     }
   },
+  "amenity/restaurant|Pizza Paï": {
+    "matchTags": ["amenity/fast_food"],
```

Could be either a fast-food, either a restaurant.  https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md#nomatch
```suggestion
    "nomatch": ["amenity/fast_food|Pizza Paï"],
```

---

and a similar entry should be added in [brands/amenity/fast_food.json](https://github.com/osmlab/name-suggestion-index/blob/master/brands/amenity/fast_food.json)